eks_infrastructure/patch_hosts.json

```diff
@@ -0,0 +1,11 @@
+{
+    "contexts": [
+        "PR",
+        "MAINLINE"
+    ],
+    "eks_clusters": [
+        "dlc-pytorch",
+        "dlc-tensorflow"
+    ],
+    "eks_version": "1.25"
```

Comment:
lets remove `eks_version` from this file. We should refer the value of `eks_version` from `build_param.json` since it is a source of truth of of the latest eks version.